### PR TITLE
Remove some reflection calls in favor of direct calls

### PR DESCRIPTION
**Pull request by Taranchuk**
## Additions
- No new functionality

## Changes
- Replaced some slow reflection calls with direct calls instead, which should be faster
- Replaced Wiri's publicizer in favor of Kraf's publicizer so it won't mess with external files. Also much less dependency - 5 nuget packages squashed to 3 nuget packages.
## Reasoning
I noticed one high consuming patch on PawnRenderer Head mat method and it turned out that it used a slow invoke call. I replaced it and then I searced for other slow reflection calls and replaced them 
## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

**Comment by Taranchuk** [comment]
Replaced Wiri's publicizer in favor of Kraf's publicizer so it doesn't mess with external files

**Comment by simplyWiri** [comment]
@perkinslr Would this change require a change to your build script as well? - If so, I think this should be your call.

**Comment by perkinslr** [comment]
> @perkinslr Would this change require a change to your build script as well? - If so, I think this should be your call.

Hm, it fails to build now that the smarter Make.py is in play.  Can we split it in two?  Do the .csproj changes alone to see if that is the source of the issue?  

Oh, on closer look, I see what the issue is.  Should be easy enough to make the build script smart enough to handle this `Publicize` directive format.

**Comment by Taranchuk** [comment]
I have removed dll and pdb files, so it can be merged now

**Review by N7Huntsman — APPROVED** [review]
Works in playtesting without any issues. Has my approval so long as Perkins is satisfied with it.

**Comment by Taranchuk** [comment]
> whereas we just didn't want the rebuilt version from you side

Why not add these files to .gitignore? So they won't be pushed to the repo?

**Pull request closed without merging**
